tests/conftest.py

```diff
@@ -87,6 +92,46 @@ def values_str() -> TListValuesInputStr:
     return [uuid_str(), uuid_str(), uuid_str()]
 
 
+@pytest.fixture
+def dictionary_name() -> TDictionaryName:
+    return uuid_str()
+
+
+@pytest.fixture
+def dictionary_field() -> TDictionaryField:
+    return uuid_str()
+
+
+@pytest.fixture
+def dictionary_value() -> TDictionaryValue:
+    return uuid_str()
+
+
+@pytest.fixture
+def dictionary_field_bytes() -> TDictionaryField:
+    return uuid_bytes()
+
+
+@pytest.fixture
+def dictionary_value_bytes() -> TDictionaryValue:
```

Comment:
```suggestion
def dictionary_value_bytes() -> TDictionaryValueBytes:
```

Reply:
Fixed in 948e864. I used `bytes` since it's in the name anyway and seems overkill to introduce a new type alias.